libipl/libipl.C

```diff
@@ -359,3 +362,42 @@ void ipl_error_callback(const ipl_error_info& error)
 		return;
 	g_ipl_error_callback_fn(error);
 }
+
+void ipl_process_error(const fapi2::ReturnCode& fapirc, 
+		struct pdbg_target *target, bool deconfig)
+{
+	if(fapirc == fapi2::FAPI2_RC_SUCCESS) {
+		ipl_error_callback(IPL_ERR_OK);
+	} 
+	else if(fapirc.getCreator() == fapi2::ReturnCode::CREATOR_PLAT) {
```

Comment:
In all the PLAT error we don't want to add callout , only for harwadre errors need to add.
libekb plat code should add custom plat error type for harwdare failures.

Reply:
Can you add some helper function for this `else if` block?

Reply:
prefer helper functions if the code is used in multiple places, we loose the clairty if we jump from one place to other.

Reply:
No, IMO, it will clean to know what we are doing inside the block statement if adding more codes.